generic_config_updater/gcu_table_modification_validators.conf.json

```diff
@@ -0,0 +1,39 @@
+{
+    "README": [
+        "table_modification_validators provides, module & method name as ",
+        "    <module name>.<method name>",
+        "NOTE: module name could have '.'",
+        " ",
+        "The last element separated by  '.' is considered as ",
+        "method name",
+        "",
+        "e.g. 'show.acl.test_acl'",
+        "",
+	"table_modification_validators_and for a given table defines a list of validators that all must pass for modification to the table to be allowed",
+	"table_modification_validators_or for a given table defines a list of validators, at least one of which must pass for modification of the table to be allowed",
+        ""
+    ],
+    "tables": {
+	"PFC_WD": {
+            "table_modification_validators_and": [ "generic_config_updater.validators.is_mellanox_device_validator" ],
```

Comment:
>table_modification_validators_and [](http://example.com/codeflow?start=13&length=33)

There is no design doc for these new field `table_modification_validators_and` `table_modification_validators_or`. It is easy to understand if either of them has value, but what is the semantics if both of them appear?

Reply:
I changed back to just `table_modification_validators` 
So now, all listed validators must pass in order for modification to the table to be allowed. This is the same logic as MoveValidators, but MoveValidators checks on every intermediary move during patch sorting.